doc/getting-started/getting-started.rst

```diff
@@ -345,14 +337,14 @@ Generate a Board Configuration File
 
              mkdir -p ~/acrn-work
              disk="/media/$USER/"$(ls /media/$USER)
-             cp -r "$disk"/acrn-board-inspector*.deb  ~/acrn-work
+             cp -r "$disk"/python3-acrn-board-inspector*.deb  ~/acrn-work
 
 #. Now that we've got the Board Inspector Debian package on the target system, install it there:
 
    .. code-block:: bash
 
       cd  ~/acrn-work
-      sudo apt install -y ./acrn-board-inspector*.deb
+      sudo apt install -y ./python3-acrn-board-inspector*.deb
```

Comment:
I got this permission denied error when trying to install the board-inspector debian package.  Is this a fatal error?:
```
acrn@acrn-Vecow:~/acrn-work$ sudo apt install ./python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'python3-acrn-board-inspector' instead of './python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb'
The following additional packages will be installed:
  cpuid msr-tools python3-bs4 python3-elementpath python3-html5lib
  python3-lxml python3-soupsieve python3-webencodings python3-xmlschema
Suggested packages:
  acrn-doc python3-genshi python-lxml-doc
The following NEW packages will be installed:
  cpuid msr-tools python3-acrn-board-inspector python3-bs4 python3-elementpath
  python3-html5lib python3-lxml python3-soupsieve python3-webencodings
  python3-xmlschema
0 upgraded, 10 newly installed, 0 to remove and 2 not upgraded.
Need to get 1,771 kB/1,868 kB of archives.
After this operation, 8,585 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 /home/acrn/acrn-work/python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb python3-acrn-board-inspector all 3.2-unstable~1.gbp808b4f [97.1 kB]
Get:2 http://us.archive.ubuntu.com/ubuntu jammy/universe amd64 cpuid amd64 20211210-1 [104 kB]
Get:3 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 msr-tools amd64 1.3-4 [10.3 kB]
Get:4 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-lxml amd64 4.8.0-1build1 [1,150 kB]
Get:5 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-elementpath all 2.5.0-1 [123 kB]
Get:6 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-xmlschema all 1.10.0-1 [172 kB]
Get:7 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-soupsieve all 2.3.1-1 [33.0 kB]
Get:8 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-bs4 all 4.10.0-2 [79.1 kB]
Get:9 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-webencodings all 0.5.1-4 [11.8 kB]
Get:10 http://us.archive.ubuntu.com/ubuntu jammy/main amd64 python3-html5lib all 1.1-3 [87.0 kB]
Fetched 1,771 kB in 1s (1,622 kB/s)       
Selecting previously unselected package cpuid.
(Reading database ... 198279 files and directories currently installed.)
Preparing to unpack .../0-cpuid_20211210-1_amd64.deb ...
Unpacking cpuid (20211210-1) ...
Selecting previously unselected package msr-tools.
Preparing to unpack .../1-msr-tools_1.3-4_amd64.deb ...
Unpacking msr-tools (1.3-4) ...
Selecting previously unselected package python3-lxml:amd64.
Preparing to unpack .../2-python3-lxml_4.8.0-1build1_amd64.deb ...
Unpacking python3-lxml:amd64 (4.8.0-1build1) ...
Selecting previously unselected package python3-elementpath.
Preparing to unpack .../3-python3-elementpath_2.5.0-1_all.deb ...
Unpacking python3-elementpath (2.5.0-1) ...
Selecting previously unselected package python3-xmlschema.
Preparing to unpack .../4-python3-xmlschema_1.10.0-1_all.deb ...
Unpacking python3-xmlschema (1.10.0-1) ...
Selecting previously unselected package python3-acrn-board-inspector.
Preparing to unpack .../5-python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_
all.deb ...
Unpacking python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
Selecting previously unselected package python3-soupsieve.
Preparing to unpack .../6-python3-soupsieve_2.3.1-1_all.deb ...
Unpacking python3-soupsieve (2.3.1-1) ...
Selecting previously unselected package python3-bs4.
Preparing to unpack .../7-python3-bs4_4.10.0-2_all.deb ...
Unpacking python3-bs4 (4.10.0-2) ...
Selecting previously unselected package python3-webencodings.
Preparing to unpack .../8-python3-webencodings_0.5.1-4_all.deb ...
Unpacking python3-webencodings (0.5.1-4) ...
Selecting previously unselected package python3-html5lib.
Preparing to unpack .../9-python3-html5lib_1.1-3_all.deb ...
Unpacking python3-html5lib (1.1-3) ...
Setting up msr-tools (1.3-4) ...
Setting up cpuid (20211210-1) ...
Setting up python3-webencodings (0.5.1-4) ...
Setting up python3-elementpath (2.5.0-1) ...
Setting up python3-html5lib (1.1-3) ...
Setting up python3-xmlschema (1.10.0-1) ...
Setting up python3-lxml:amd64 (4.8.0-1build1) ...
Setting up python3-soupsieve (2.3.1-1) ...
Setting up python3-bs4 (4.10.0-2) ...
Setting up python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
Processing triggers for man-db (2.10.2-1) ...
N: Download is performed unsandboxed as root as file '/home/acrn/acrn-work/python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb' couldn't be accessed by user '_apt'. - pkgAcquire::Run (13: Permission denied)
```

Reply:
After a bit of research, I found a description of this problem saying it's because the user _apt does not have access to the .deb file.  I made a copy of that file into the /tmp directory and tried installing it from there and it worked just fine:
```
acrn@acrn-Vecow:~/acrn-work$ cp python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb /tmp
acrn@acrn-Vecow:~/acrn-work$ cd /tmp
acrn@acrn-Vecow:/tmp$ sudo apt remove python3-acrn-board-inspector
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cpuid msr-tools python3-bs4 python3-elementpath python3-html5lib python3-lxml python3-soupsieve python3-webencodings python3-xmlschema
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED:
  python3-acrn-board-inspector
0 upgraded, 0 newly installed, 1 to remove and 2 not upgraded.
After this operation, 627 kB disk space will be freed.
Do you want to continue? [Y/n] y
(Reading database ... 198723 files and directories currently installed.)
Removing python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
acrn@acrn-Vecow:/tmp$ sudo apt install ./python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'python3-acrn-board-inspector' instead of './python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb'
Suggested packages:
  acrn-doc
The following NEW packages will be installed:
  python3-acrn-board-inspector
0 upgraded, 1 newly installed, 0 to remove and 2 not upgraded.
Need to get 0 B/97.1 kB of archives.
After this operation, 627 kB of additional disk space will be used.
Get:1 /tmp/python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb python3-acrn-board-inspector all 3.2-unstable~1.gbp808b4f [97.1 kB]
Selecting previously unselected package python3-acrn-board-inspector.
(Reading database ... 198641 files and directories currently installed.)
Preparing to unpack .../python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb ...
Unpacking python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
Setting up python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
```

Reply:
After a bit of research, I found a description of this problem saying it's because the user _apt does not have access to the .deb file.  I made a copy of that file into the /tmp directory and tried installing it from there and it worked without an error:
```
acrn@acrn-Vecow:~/acrn-work$ cp python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb /tmp
acrn@acrn-Vecow:~/acrn-work$ cd /tmp
acrn@acrn-Vecow:/tmp$ sudo apt remove python3-acrn-board-inspector
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cpuid msr-tools python3-bs4 python3-elementpath python3-html5lib python3-lxml python3-soupsieve python3-webencodings python3-xmlschema
Use 'sudo apt autoremove' to remove them.
The following packages will be REMOVED:
  python3-acrn-board-inspector
0 upgraded, 0 newly installed, 1 to remove and 2 not upgraded.
After this operation, 627 kB disk space will be freed.
Do you want to continue? [Y/n] y
(Reading database ... 198723 files and directories currently installed.)
Removing python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
acrn@acrn-Vecow:/tmp$ sudo apt install ./python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'python3-acrn-board-inspector' instead of './python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb'
Suggested packages:
  acrn-doc
The following NEW packages will be installed:
  python3-acrn-board-inspector
0 upgraded, 1 newly installed, 0 to remove and 2 not upgraded.
Need to get 0 B/97.1 kB of archives.
After this operation, 627 kB of additional disk space will be used.
Get:1 /tmp/python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb python3-acrn-board-inspector all 3.2-unstable~1.gbp808b4f [97.1 kB]
Selecting previously unselected package python3-acrn-board-inspector.
(Reading database ... 198641 files and directories currently installed.)
Preparing to unpack .../python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb ...
Unpacking python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
Setting up python3-acrn-board-inspector (3.2-unstable~1.gbp808b4f) ...
```

Reply:
Though this permission denied message is related to bug [#1543280](https://bugs.launchpad.net/ubuntu/+source/aptitude/+bug/1543280) and has no impact to the function of acrn-board-inspector, I will update the doc to put the acrn-board-inspector deb package into /tmp folder of the target board to avoid this error message which could confuse users. Is that OK?

Reply:
That particular Ubuntu bug (#1543280) was reported in 2016 (6 years ago).  In the v3.1 GSG we also put the board inspector debian package in the ~/acrn-work folder on the target and had no problem installing it.  Why do we have a problem with the new Debian packaging solution?

Reply:
I can reproduce this issue on Ubuntu 22.04 but can't reproduce it on Ubuntu 20.04 with the 2 deb packages built by the new debuild method and old method `make board_inspector`. This is a known issue as described in https://docs.docker.com/desktop/install/ubuntu/#install-docker-desktop. Since we use Ubuntu 20.04 in v3.1, we don't see this issue on v3.1 GSG.